tests/console/sle15_workarounds.pm

```diff
@@ -18,11 +18,13 @@
 use base qw(consoletest distribution);
 use strict;
 use testapi;
-use utils qw(zypper_call sle_version_at_least);
+use utils qw(zypper_call sle_version_at_least pkcon_quit);
 
 
 sub run {
     my $self = shift;
+    # Stop packagekit
+    pkcon_quit;
```

Comment:
Put this call after version check, so we do nothing for other versions even if module is selected.